lib/mpi/service.rb

```diff
@@ -111,6 +112,20 @@ def find_profile_by_edipi(edipi:, search_type: Constants::CORRELATION_WITH_RELAT
       MPI::Services::FindProfileResponseCreator.new(type: Constants::FIND_PROFILE_BY_EDIPI_TYPE, error: e).perform
     end
 
+    def find_profile_by_facility(facility_num:, dfn:, search_type: Constants::CORRELATION_WITH_RELATIONSHIP_DATA)
```

Comment:
do you mind expanding the name of this param to `facility_number`? I prefer when it's verbose, I think it looks better in Ruby that way

Reply:
Could we expand `dfn`? Would be nice to have something more descriptive for this field. Is it data file number? Could make the param `data_file_number`

Reply:
@bosawt Can do!  I'm thinking it might make since to name it `facility_id` actually to remain consistent with our id parsing constants

`vha_facility_ids: { regex: VHA_FACILITY_IDS_REGEX, root_oid: VA_ROOT_OID, type: :facility }`

Reply:
@bosawt As for `dfn`, I have yet to hear an expansion on the acronym though `data_file_number` would make the most sense. I believe OIT refers to it as vista ID as well as DFN so it may be more clear to just name it vista id

Saw this comment from a conversation with John Bramley when he worked on the `vha_facility_hash` parsing work

> the vha_facility_hash has the facility ID as the key and then an array of vista IDs for that facility as the values